src/test/java/duke/parser/ParserTest.java

```diff
@@ -0,0 +1,45 @@
+package duke.parser;
+
+import duke.DukeException;
+import duke.command.*;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ParserTest {
+    @Test
+    public void parse_ValidInputs_CreateCommands() {
```

Comment:
According to the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html), test methods should also be in camelCase
```suggestion
    public void parse_validInputs_createCommands() {
```